cephfs/file_futime.go

```diff
@@ -0,0 +1,93 @@
+//go:build ceph_preview
+
+package cephfs
+
+/*
+#cgo LDFLAGS: -lcephfs
+#cgo CPPFLAGS: -D_FILE_OFFSET_BITS=64
+#include <errno.h>
+#include <stdlib.h>
+#include <cephfs/libcephfs.h>
+*/
+import "C"
+
+import (
+	ts "github.com/ceph/go-ceph/internal/timespec"
+)
+
+// Futime changes file/directory last access and modification times.
+//
+// Implements:
+//
+//	int ceph_futime(struct ceph_mount_info *cmount, int fd, struct utimbuf *buf);
+func (f *File) Futime(times *Utime) error {
+	if err := f.validate(); err != nil {
```

Comment:
Why not `return Futime(f.mount, f.fd)` ?
That'd accomplish the same thing with less copy-paste and chance of the two implementations getting out of sync. If you're thinking about eventually deprecating the other calls, we can handle that in the future by "swapping" the implementations. I'm just not keen on the duplication.